weave/guides/core-types/media.mdx

```diff
@@ -3,53 +3,59 @@ title: "Log media"
 description: "Log media returned in your traces, such as images and videos."
 ---
 
-W&B Weave supports logging and has dedicated displays for numerous content types such as videos, images, audio files, PDFs, CSVs and HTML.
+W&B Weave supports logging and has dedicated displays for numerous content types such as videos, images, audio files, PDFs, CSVs and HTML. Each section of this guide provides a basic configuration example and a more advanced example of how to log the media type.
```

Comment:
```suggestion
W&B Weave supports logging and displaying numerous content types, with dedicated functions for displaying videos, images, audio clips, PDFs, CSV data, and HTML. This guide provides basic and advanced examples for logging and displaying each media type.
```
Not sure if this is what you actually mean. I'm also not sure that the manual TOC is necessary, since the right-hand TOC will have the same structure.

Reply:
nope, this is much better! Thank you